HW3/Exceptions/DuplicateUserCredentialsException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace HW3.Exceptions
+{
+    internal class DuplicateUserCredentialsException : Exception
+    {
+        public DuplicateUserCredentialsException(string? message) : base(message)
```

Comment:
Do you know the difference betwen `string` and `string?`

Reply:
Good point! Yes, I know the difference between string and string?... string? is nullable.